### PR TITLE
feat: provide composed css variables

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: composed-variables/composed-variables.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
+:root {
```
Comment (jeripeierSBB) [inline composed-variables/composed-variables.css:1]: Comment with purpose of the file

File: composed-variables/composed-variables.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+  /* Micro */
+  @media (min-width: 22.5rem) {
+    /* Layout */
+    --sbb-layout-base-offset-responsive: var(--sbb-layout-base-offset-responsive-micro);
+    --sbb-grid-base-columns: var(--sbb-layout-base-grid-columns-micro);
+    --sbb-grid-base-gutter-responsive: var(--sbb-layout-base-grid-gutter-responsive-micro);
+  }
+
+  /* Small */
```
Comment (jeripeierSBB) [inline composed-variables/composed-variables.css:62]: comment values with css variables

File: composed-variables/composed-variables.css (hunk 1 above)
Comment (jeripeierSBB) [inline composed-variables/composed-variables.css:1]: start and ending comment instead (for regex)

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**